cpp/CMakeLists.txt

```diff
@@ -131,7 +137,7 @@ function(bbp_enable_static_analysis)
   set(${PROJECT_NAME}_ClangTidy_OPTIONS -extra-arg=-Wno-unknown-warning-option
       CACHE STRING "clang-tidy command options")
   mark_as_advanced(${PROJECT_NAME}_ClangTidy_OPTIONS)
-  set(${PROJECT_NAME}_ClangTidy_FILES_RE "^.*\\\\.c$$"  "^.*\\\\.cpp$$"
+  set(${PROJECT_NAME}_ClangTidy_FILES_RE "^.*\\\\.c$$" "^.*\\\\.cpp$$"
```

Comment:
I have contradictory feelings about it On one side, `.cpp` is the only extension supported by BBP coding conventions, and as a user it would surprise me other extensions were taken into account by default. It is still possible to override this variable so it is not a blocker (see documentation). 

On the other side, .cc and .cxx are pretty common, and they should be taken into account by default.